CRM/Upgrade/Incremental/php/FiveFortyFour.php

```diff
@@ -49,27 +49,16 @@ public function setPostUpgradeMessage(&$postUpgradeMessage, $rev): void {
     // }
   }
 
-  /*
-   * Important! All upgrade functions MUST add a 'runSql' task.
-   * Uncomment and use the following template for a new upgrade version
-   * (change the x in the function name):
+  /**
+   * Upgrade function.
+   * @param string $rev
    */
-
-  //  /**
-  //   * Upgrade function.
-  //   *
-  //   * @param string $rev
-  //   */
-  //  public function upgrade_5_0_x($rev): void {
-  //    $this->addTask(ts('Upgrade DB to %1: SQL', [1 => $rev]), 'runSql', $rev);
-  //    $this->addTask('Do the foo change', 'taskFoo', ...);
-  //    // Additional tasks here...
-  //    // Note: do not use ts() in the addTask description because it adds unnecessary strings to transifex.
-  //    // The above is an exception because 'Upgrade DB to %1: SQL' is generic & reusable.
-  //  }
-
-  // public static function taskFoo(CRM_Queue_TaskContext $ctx, ...): bool {
-  //   return TRUE;
-  // }
+  public function upgrade_5_44_alpha1($rev) {
+    // The runSql task populates the new column, so this addColumn task runs first
+    $this->addTask('Add case_id column to civicrm_relationship_cache', 'addColumn',
+      'civicrm_relationship_cache', 'case_id', "int unsigned DEFAULT NULL COMMENT 'FK to civicrm_case'"
```

Comment:
@colemanw on upgrade your just adding the column not the FK but on install it would have an additional FK right based on the xml?

Reply:
Ugh, how do we even do that?

Reply:
https://github.com/civicrm/civicrm-core/blob/09e790b7a8ad75c107ff6eb9cf0f0f6f93856956/CRM/Upgrade/Incremental/php/FiveForty.php#L93